checkstyle-tester/checks-nonjavadoc-error.xml

```diff
@@ -298,7 +298,9 @@
     <module name="IllegalTokenText"/>
     <module name="IllegalType"/>
     <module name="InnerAssignment"/>
-    <module name="MagicNumber"/>
+    <module name="MagicNumber">
```

Comment:
No, we need both. The other has the value of `false` correct? They will be testing different things then.